src/components/EventList.spec.tsx

```diff
@@ -19,8 +20,23 @@ describe('EventList', () => {
             });
     });
 
+    const withIntl = (
+        events : ZetkinEvent[] | undefined,
+        org : ZetkinOrganization) => {
+        return (
+            <IntlProvider
+                locale="en"
+                messages={{
+                    'components.eventList.placeholder':
+                    'Sorry, there are no planned events at the moment.',
+                }}>
+                <EventList events={ events } org={ org }/>
+            </IntlProvider>
+        );
+    };
```

Comment:
I'm not a fan of how this function obfuscates from the actual tests (below) how `<EventList>` is used, by passing in props as arguments.

I think this should either be implemented the same that `mountWithTheme()` is in `RegistrationForm.spec.tsx`, or as a reusable higher-order function. In both cases, the  `<EventList events={ events } org={ org }/>` part would be part of the `it()` test cases instead of hidden away in this function.

The function should require no arguments, unless it's implemented as a higher-order function factory where messages/locale can be passed into the factory.